<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Structured Light Suite</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>535</width>
     <height>530</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Holoprojector</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="mainView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="ViewController" name="viewController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="EncoderController" name="encodeController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="CaptureController" name="captureController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="fileList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>512</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRotate"/>
   <addaction name="actionZoom"/>
   <addaction name="actionPan"/>
  </widget>
  <widget class="QToolBar" name="modeBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="modeEncode"/>
   <addaction name="modeCapture"/>
   <addaction name="modeView"/>
  </widget>
  <widget class="QStatusBar" name="infoBar"/>
  <action name="actionOpenXYZM">
   <property name="text">
    <string>Open XYZ</string>
   </property>
  </action>
  <action name="actionOpen_Holovideo">
   <property name="text">
    <string>Open Holovideo</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/images/Orbit Tool</normaloff>:/images/Orbit Tool</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/images/Zoom Tool</normaloff>:/images/Zoom Tool</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>zoom</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/images/Pan Tool</normaloff>:/images/Pan Tool</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="toolTip">
    <string>pan</string>
   </property>
  </action>
  <action name="actionOpen_Holoimage">
   <property name="text">
    <string>Open Holoimage</string>
   </property>
  </action>
  <action name="actionExportSingleFrame">
   <property name="text">
    <string>Single Frame</string>
   </property>
  </action>
  <action name="actionExportEntireVideo">
   <property name="text">
    <string>Entire Video</string>
   </property>
  </action>
  <action name="modeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View Geometry</string>
   </property>
  </action>
  <action name="modeEncode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Encode</string>
   </property>
   <property name="toolTip">
    <string>Encode Geometry</string>
   </property>
  </action>
  <action name="modeCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Capture Geometry</string>
   </property>
  </action>
  <action name="actionCalibrateRefrence">
   <property name="text">
    <string>Calibrate Reference</string>
   </property>
  </action>
  <action name="actionConnectCamera">
   <property name="text">
    <string>Connect Camera</string>
   </property>
  </action>
  <action name="actionDisconnectCamera">
   <property name="text">
    <string>Disconnect Camera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewController</class>
   <extends>QWidget</extends>
   <header>ViewController.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EncoderController</class>
   <extends>QWidget</extends>
   <header>Encode/EncoderController.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptureController</class>
   <extends>QWidget</extends>
   <header location="global">Capture/CaptureController.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
